client/homebrew/pages/basePages/listPage/brewItem/brewItem.jsx

```diff
@@ -98,6 +98,12 @@ const BrewItem = createClass({
 		const dateFormatString = 'YYYY-MM-DD HH:mm:ss';
 
 		return <div className='brewItem'>
+			{brew.thumbnail ?
+				<div className='thumbnail'>
+					<img className='thumbnailImage' src={brew.thumbnail} ></img>
+				</div> :
+				<></>
```

Comment:
Do we need this empty fragment?

Reply:
I think you can do something like 

```
{brew.thumbnail && <div ....
```

So it draws the div if true, and just doesn't do anything if false.